assets/i18n/en.json

```diff
@@ -445,7 +445,7 @@
     "I18N_SPLASH_SUBTITLE": "Oppia makes it easy to create interactive lessons that educate and engage.",
     "I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": "Oppia lets you <a href=\"/creator_dashboard?mode=create\">create</a> and <a href=\"/library\">share</a> explorations on a wide range of subjects, limited only by your imagination.",
     "I18N_SPLASH_TITLE": "Think outside the books.",
-    "I18N_START_HERE": "Start Here!",
+    "I18N_START_HERE": "Click here to start",
```

Comment:
How about this?
![image](https://user-images.githubusercontent.com/29590798/28995637-72ef9d14-7a42-11e7-93d6-aab51ed73394.png)